packages/components/scss/new_variables.scss

```diff
@@ -6,7 +6,7 @@ $spacer-3: map-get($spacers, 3);
 $spacer-4: map-get($spacers, 4);
 $spacer-5: map-get($spacers, 5);
 
-/* Marching Ants for golden layout dropzone and drag and drop */
+//Marching Ants for golden layout dropzone and drag and drop
```

Comment:
I changed this because when I was inspecting the styles outputted, this comment was in every <style> tag in <head>. I don't know why it includes it if it's in `/* */`, but I just changed it to // to be consistent with the rest of the comment and so it doesn't appear it _every_ style tag.